components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/user/OpenIDConnectUserEndpoint.java

```diff
@@ -55,6 +56,7 @@
 
     @GET
     @Path("/")
+    @Consumes("application/x-www-form-urlencoded")
```

Comment:
Need to check whether this would break existing clients. Since we do not expect a content type header with user info response